ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixParticipant.java

```diff
@@ -608,6 +611,11 @@ public boolean updateDiskCapacity(int diskCapacity) {
     return success;
   }
 
+  @Override
+  public void unblockStateTransition() {
+      this.blockStateTransitionLatch.countDown();
+  }
+
```

Comment:
Please add some comments to add more context on why we want to do this, with regard to auto registration.